### DISCUSSION TITLE
How to convert to SBC mono?

### DISCUSSION THREAD

**Discussion by leelx78** [pschatzmann/arduino-audio-tools#836] [Q&A] [answered]
Hi I'm having problems in converting output to mono. My application requires
- stereo IN
- mono OUT

My I2S codec is set to stereo 44.1kHz/16bit (both ADC and DAC) and using the SBC decoder I can send and receive packets if I use stereo for both input and output. Problem is that this way I'm wasting half the outgoing bandwidth so I have been trying to convert output to mono in 2 ways:

```
I2SStream -> ChannelFormatConverterStream -> SBC Encoder -> UDP
I2SStream -> FormatConverterStream -> SBC Encoder -> UDP
```

But no matter what I do the output is still about 320kbps. I made sure the SBC Encoder is initialised with 1ch but although there is a difference in the data, the rate is the same.

What's the proper way to achieve this? 

Thanks!

**Comment by pschatzmann** [comment, marked as answer]
Both of your proposals should work but are unnecessary! 
Just set the channels of the I2SStream to 1 and you will get a mono signal!
